Makefile

```diff
@@ -0,0 +1,81 @@
+SHELL := /bin/bash
+
+JS_FILES = $(shell find -type f -and \( -name "*.js" \))
+UI_FILES = $(shell find -type f -and \( -name "*.ui" \))
```

Comment:
These variables contain all the `js`/`ui` files in the project.
Not really relevant in this project, but nice to know:

Variable assignments made with `:=` are "fixed" - meaning they are evaluated at creation and then saved as a hardcoded string.
Variable assignments made with `=` are variable - meaning they are evaluated each time they get called in the `Makefile`.
This means: Would, during the execution of this `Makefile`, another `ui` file be created and then `UI_FILES` gets called again, the new file would be included. Does this explanation make any sense? :exploding_head:

Reply:
Thanks for the explanation! Also the "General Makefile Syntax" at the top is very helpful!